keep/api/core/db.py

```diff
@@ -1620,3 +1621,59 @@ def get_all_presets(tenant_id: str) -> List[Preset]:
             select(Preset).where(Preset.tenant_id == tenant_id)
         ).all()
     return presets
+
+
+def get_all_actions(tenant_id: str) -> List[Action]:
+    with Session(engine) as session:
+        actions = session.exec(
+            select(Action).where(Action.tenant_id == tenant_id)
+        ).all()
+    return actions
+
+
+def get_action(tenant_id: str, action_id: str) -> Action:
+    with Session(engine) as session:
+        action = session.exec(
+            select(Action).where(Action.tenant_id == tenant_id).where(Action.id == action_id)
+        ).first()
+    return action
+
+
+def create_action(action: Action):
+    with Session(engine) as session:
+        session.add(action)
+        session.commit()
+        session.refresh(action)
+
+
+def create_actions(actions: List[Action]):
+    with Session(engine) as session:
+        for action in actions:
+            session.add(action)
+        session.commit()
+
+
+def delete_action(tenant_id: str, action_id: str) -> Union[Action, None]:
+    with Session(engine) as session:
+        found_action = session.exec(
+            select(Action).where(Action.id == action_id).where(Action.tenant_id == tenant_id)
+        ).first()
+        if found_action:
+            session.delete(found_action)
+            session.commit()
+            return found_action
```

Comment:
I would return True/False because the found_action is deleted and shouldn't be use